tests/legacy-cli/e2e/utils/assets.ts

```diff
@@ -39,10 +39,12 @@ export function copyAssets(assetName: string) {
 }
 
 
-export function createProjectFromAsset(assetName: string, useNpmPackages = false) {
-  return Promise.resolve()
-    .then(() => copyAssets(assetName))
-    .then(dir => process.chdir(dir))
-    .then(() => useNpmPackages ? null : useBuiltPackages())
-    .then(() => silentNpm('install'));
+export async function createProjectFromAsset(assetName: string, useNpmPackages = false) {
+  const dir = await copyAssets(assetName);
+  process.chdir(dir);
+  if (!useNpmPackages) {
+    await useBuiltPackages();
```

Comment:
Technically, no test should be using the built packages directly anymore but this fixes the problem for now.

We should revisit these tests though.  Potentially using `npx` to generate a new project of the appropriate version directly.